Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -181,15 +181,7 @@ + (NSInteger)getRetryCount {
 }
 
 + (NSString *)getAppKey {
-    NSString *ret = [[[NSBundle mainBundle] infoDictionary] objectForKey:KEY_APP_KEY];
-    if (!ret || ret.length == 0) {
-        // for backward compatibility
-        ret = [BNCPreferenceHelper readStringFromDefaults:KEY_APP_KEY];
```

Comment:
I'm talking in a hypothetical situation.

Say an early adopter used a version of Branch that saved to defaults with key "asdf."
In a later version, the removed the key "asdf," but didn't add a new key.
When that app is released, new users would have no key, old users would incidentally still get the "asdf" key.

That's the scenario I meant -- it was the same w/ the `setRetryInterval` methods and such that were removed a few versions back.